user-guide/Advanced_Functionality/DataMiner_Agents/Installing_a_DMA/Installing_a_DMA_FAQ.md

```diff
@@ -0,0 +1,366 @@
+
+1. In the left pane of the IIS Manager, open the website and verify that the Dashboards item is an application (cogwheel icon).
+
+   If it is not, proceed as follows:
+
+   1. Right-click > *Properties*
+   1. Click *Create Application*.
+   1. On the *Documents* tab, make sure that the default document is "default.aspx".
+   1. On the *Directory Security* tab, click *Edit* (next to *Authentication and access control*), and make sure that "Integrated Windows Authentication" is selected.
+
+1. Restart IIS:
+
+   - In the left pane, right-click COMPUTERNAME and select *All Tasks > Restart IIS*.
+   - Alternatively, you can run the `iisreset` command from a command prompt.
+
+## What do I do if there is an error when the MySQL service is started?
+
+After installing MySQL, the following error may occur when the MySQL service attempts to start:
+
+```txt
+MySQL service cannot start (event viewer error message: Default storage engine <InnoDB> is not available)
+```
+
+To resolve this issue, proceed as follows:
+
+1. Go to the MySQL Server installation directory:
+
+   - On an x86 machine: *C:\Program Files\MySQL\MySQL Server x.x\Data*
+   - On an x64 machine: *C:\Program Files(86)\MySQL\MySQL Server x.x\Data*
+
+1. Remove the following files:
+
+   - ib_logfile0
+   - ib_logfile1
+   - ibdata1
+
+1. Start the MySQL service manually.
+
+For more information, see <http://forums.mysql.com/read.php?22,58270,122989#msg-122989>.
+
+## What do I do if I get a 'Trust not granted' error when opening Cube?
+
+Install the necessary certificates.
+
+For more information, see <http://[DMA]/tools/>.
+
+## What do I do if I get an 'unauthorized access exception' after saving annotations?
+
+On Windows 7, the following error may occur after saving annotations:
+
+```txt
+SYSTEM.UNAUTHORIZEDACCESSEXCEPTION: RETRIEVING THE COM CLASS FACTORY FOR COMPONENT WITH CLSID...
+```
+
+This is because the IIS_IUSRS group has not been granted "Local Activation" permission on SLDMS.
+
+> [!TIP]
+> See also: [How do I manually configure DCOM?](#how-do-i-manually-configure-dcom)
+
+## How do I manually configure Windows Firewall?
+
+Normally, Windows Firewall is configured automatically when you install DataMiner. However, if you want to configure it manually, do the following:
+
+1. Go to *Start > Control Panel > Windows Firewall*.
+
+1. Click *Allow an app or feature through Windows Firewall*.
+
+1. Click the *Allow another app...* button.
+
+1. Next to *Path*, click the *Browse* button and browse to *C:\Program Files (x86)\Internet Explorer\ieexplore.exe*.
+
+1. Click *Add*.
+
+1. Go back to the *Windows Firewall* page of the control panel and click *Advanced Settings*.
+
+1. On the *Inbound Rules* tab page, the following ports will need to be added:
+
+   |Name | Port number | Protocol |
+   |-----| ----------- | -------- |
+   | HTTP Port | 80 | TCP |
```

Comment:
From 10.2.X we will also support HTTPS by default so then 443 should be opened as well